compose.yml

```
@@ -0,0 +1,49 @@
+
+  app:
+    build: .
+    depends_on:
+      - kafka
+    environment:
+      KAFKA_BOOTSTRAP_SERVERS: kafka:9092
+      KAFKA_GROUP_ID: my_consumer_group
+    develop:
+      watch:
+        - path: .
+          patterns: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)